tests/test_healthcheck.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime
+from flask import url_for
+from mock import Mock, patch
+from redis import ConnectionError
+from tests import TestCase
+
+from portal.views.healthcheck import (
+    celery_available,
+    celery_beat_available,
+    get_celery_beat_threshold,
+    postgresql_available,
+    redis_available,
+)
+
+
+class TestHealthcheck(TestCase):
```

Comment:
I generally don't like writing unit tests that test implementation, but I decided to go this route so I wouldn't have to start and stop processes like redis, celery, and postgresql. Please share thoughts if you have them. Do you think there's a better way to test this? My main goal was to make sure the functions I wrote behave as expected given expected values from dependent services.